packages/pointer_interceptor/pointer_interceptor_ios/example/integration_test/pointer_interceptor_test.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('placeholder test', (WidgetTester tester) async {});
```

Comment:
Can we remove this? Does CI complain if we remove this?

Reply:
Yes, I added it back because it fails if theres no integration_test